doc/source/whatsnew/v0.18.1.txt

```diff
@@ -22,6 +22,14 @@ New features
 Enhancements
 ~~~~~~~~~~~~
 
+- Timestamps accept positional parameters like `datetime.datetime`:
+
```

Comment:
put ``Timestamp`` in double back-ticks 

say: can now accept

put a link in for ``datetime.datetime`` back to the constructor (on the python documentation page)

@jorisvandenbossche do we have a nicer way of doing this (as opposed to a straight link), IIRC, sphinx allows us to use an intra-module link (like we do with numpy)